rust/connlib/tunnel/src/client.rs

```diff
@@ -1779,12 +1757,8 @@ mod proptests {
 
     #[test_strategy::proptest]
     fn setting_resource_offline_doesnt_set_all_related_resources_offline(
```

Comment:
@conectado This test is now failing. Can you explain to me what you wanted to test here? As far as I can see from the implementation, this test should never pass? We set all sites to offline so naturally, all resources that share this site are offline too?

Reply:
This tests is for resources that share some but not all sites, therefore setting the resource offline shouldn't set the other resources offline as they have other sites through which they might be able to connect.

The naming for this tests is really bad, sorry!

The one below, `setting_resource_offline_set_all_resources_sharing_all_groups_offline` is the case where setting the resource offline will change the status of all related resources.

Reply:
> This tests is for resources that share some but not all sites, therefore setting the resource offline shouldn't set the other resources offline as they have other sites through which they might be able to connect.
> 
> The naming for this tests is really bad, sorry!
> 
> The one below, `setting_resource_offline_set_all_resources_sharing_all_groups_offline` is the case where setting the resource offline will change the status of all related resources.

We don't have multi-site resources right? So we can never hit this.

Until we properly define, what a multi-site resource is, I'd really like to avoid writing / having code that is trying to handle them.